docker/rskj/node.conf

```diff
@@ -112,3 +113,5 @@ crypto {
     # - "native" (Native C lib)
     library: "bc"
 }
+
+# transaction.accountTxRateLimit.enabled = false
```

Comment:
Why do we need this? Is it just a note for future (considering it's a comment)?

Reply:
Sorry, I just need to uncomment it, good catch!